<!DOCTYPE html>
<!-- saved from url=(0034)https://www.stat.ubc.ca/~bouchard/ -->
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="Research home page">
    <meta name="author" content="Alexandre Bouchard-Côté">

    <title>Alexandre Bouchard-Côté</title>

    <!-- Bootstrap core CSS -->
    <link href="./larkz_files/bootstrap.min.css" rel="stylesheet">

    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
    <link href="./larkz_files/ie10-viewport-bug-workaround.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="./larkz_files/jumbotron-narrow.css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>

  <body>

    <div class="container">
    
      <div class="header clearfix">
        <nav>
          <ul class="nav nav-pills pull-right">
            <li role="presentation" class="active">
              <a href="https://www.stat.ubc.ca/~bouchard/index.html">Home</a>
            </li>
            <li role="presentation">
              <a href="https://www.stat.ubc.ca/~bouchard/papers.html">Papers</a>
            </li>
            <li role="presentation">
              <a href="https://www.stat.ubc.ca/~bouchard/group.html">Group</a>
            </li>
            <li role="presentation">
              <a href="https://www.stat.ubc.ca/~bouchard/teaching.html">Teaching</a>
            </li>
            <li role="presentation">
              <a href="https://www.stat.ubc.ca/~bouchard/misc.html">Misc</a>
            </li>
          </ul>
        </nav>
        <h3 class="text-muted">Alexandre Bouchard-Côté</h3>
      </div>
      
      <div class="row marketing">
        <div class="col-lg-4">
          <img src="./larkz_files/bouchard.jpeg" class="img-rounded img-responsive" style="max-width: 200px;">
        </div>
        <div class="col-lg-8">
          <h4>Associate Professor of Statistics</h4><p><span class="glyphicon glyphicon-home"></span>&nbsp;&nbsp;University of British Columbia</p>
          <p><span class="glyphicon glyphicon-envelope"></span>&nbsp;&nbsp;bouchard@stat.ubc.ca</p>
          <p><span class="glyphicon glyphicon-map-marker"></span>&nbsp;&nbsp;<a href="https://www.google.com/maps?q=49.263125,-123.252118&amp;ll=49.263212,-123.252128&amp;spn=0.002535,0.005295&amp;num=1&amp;t=h&amp;z=18&amp;iwloc=near">ESB building, room 3124</a></p>
          <p><span class="glyphicon glyphicon-info-sign"></span>&nbsp;&nbsp;<a href="https://www.stat.ubc.ca/~bouchard/#bio">Bio</a></p>
        </div>
      </div>
      <div class="row marketing">
        <h3>Research highlights</h3>
        <hr>
        <p>
          </p><div class="row marketing">
            <div class="col-lg-4">
              <img src="./larkz_files/bps.jpg" class="img-rounded img-responsive" style="max-width: 200px;">
            </div>
            <div class="col-lg-8">
              <h4>Non-reversible Monte Carlo methods</h4>
              <p>Markov chain Monte Carlo (MCMC) is notoriously difficult to scale to problems 
              having high-dimensional latent variables ("big models"), which arise in many 
              scientific and engineering applications. 
              </p>
              <p><span class="glyphicon glyphicon-file"></span>&nbsp;&nbsp;We are developing a new perspective on <b>parallel tempering (PT) algorithms</b> and their tuning, 
                        based on two main insights. First, we identified and formalized a sharp divide in the 
                        behaviour and performance of reversible versus non-reversible PT methods. 
                        Second, we are analyzing the behaviour of PT algorithms using a novel asymptotic regime 
                        in which the number of parallel compute cores goes to infinity.
                        <a href="https://www.stat.ubc.ca/~bouchard/pub/Syed2019NRPT.pdf">Preprint</a></p>
              <p><span class="glyphicon glyphicon-file"></span>&nbsp;&nbsp;We are working on 
                                    an alternative to MCMC that we call 
                                    the "Bouncy Particle Sampler" (BPS), which imports ideas from the field of 
                                    molecular simulation to scale MCMC to high dimensional problems.  <a href="https://www.stat.ubc.ca/~bouchard/pub/Bouchard2017Bouncy.pdf">JASA paper</a></p>
              <p><span class="glyphicon glyphicon-file"></span>&nbsp;&nbsp;<a href="https://www.stat.ubc.ca/~bouchard/pub/geometric-erg-bps.pdf">Follow-up Annals of Statistics paper</a>, on the geometric ergodicity of BPS</p>
              <p><span class="glyphicon glyphicon-file"></span>&nbsp;&nbsp;<a href="https://arxiv.org/pdf/1707.05296.pdf">Preprint of follow-up work</a>, on non-linear trajectories and discrete piecewise deterministic Markov processes</p>
              <p><span class="glyphicon glyphicon-file"></span>&nbsp;&nbsp;<a href="https://arxiv.org/pdf/1905.13120.pdf">Preprint: an application of BPS to CTMC parameter estimation</a></p>
              <p><span class="glyphicon glyphicon-plus"></span>&nbsp;&nbsp;<a href="https://www.stat.ubc.ca/~bouchard/bps">More information</a></p>
            </div> 
          </div>
          <div class="row marketing">
            <div class="col-lg-4">
              <img src="./larkz_files/blang.jpg" class="img-rounded img-responsive" style="max-width: 200px;">
            </div>
            <div class="col-lg-8">
              <h4>Tools for Bayesian data science</h4>
              <p>We are developing a language and software development kit for doing Bayesian analysis. 
              The design philosophy is centered around the day-to-day requirements of real world (Bayesian) data science. 
              The inference engines brings to bear several recent advances such as non-reversible methods. 
              </p>
              <p><span class="glyphicon glyphicon-plus"></span>&nbsp;&nbsp;<a href="https://www.stat.ubc.ca/~bouchard/blang/">Project site</a></p>
            </div> 
          </div>
          <div class="row marketing">
            <div class="col-lg-4">
              <img src="./larkz_files/tree.jpg" class="img-rounded img-responsive" style="max-width: 200px;">
            </div>
            <div class="col-lg-8">
              <h4>Bayesian phylogenetic inference</h4>
              <p>As a result of advances in sequencing technologies, the fields of 
              computational and statistical phylogenetics, which are concerned with the 
              modelling and inference of evolutionary relationships, have been growing rapidly 
              in recent years. I am particularly interested in computationally-intensive 
              Bayesian methods and inference of complex evolutionary models.
              </p>
              <p><span class="glyphicon glyphicon-file"></span>&nbsp;&nbsp;<a href="https://arxiv.org/pdf/1806.08813.pdf">Prepring on a change-of-measure based phylogenetic SMC algorithm.</a></p>
              <p><span class="glyphicon glyphicon-file"></span>&nbsp;&nbsp;<a href="https://www.stat.ubc.ca/~bouchard/pub/Zhao2015Bayesian.pdf">Novel sampling method based on Hamiltonian Monte Carlo for parameter-rich evolutionary models.</a></p>
              <p><span class="glyphicon glyphicon-file"></span>&nbsp;&nbsp;<a href="https://www.stat.ubc.ca/~bouchard/pub/Zhai2017Poissonian.pdf">Long indel model</a> based on the <a href="https://www.stat.ubc.ca/~bouchard/pub/Bouchard2013Evolutionary.pdf">Poisson Indel Process.</a></p>
              <p><span class="glyphicon glyphicon-plus"></span>&nbsp;&nbsp;<a href="https://www.stat.ubc.ca/~bouchard/papers.html#Phylogenetics">More papers</a></p>
            </div> 
          </div>
          <div class="row marketing">
            <div class="col-lg-4">
              <img src="./larkz_files/ith.jpg" class="img-rounded img-responsive" style="max-width: 200px;">
            </div>
            <div class="col-lg-8">
              <h4>Probabilistic modelling of the evolutionary dynamics and phylogeny of cancer</h4>
              <p>Proliferating cancer cells, in which DNA repair mechanisms are disrupted, 
              accumulate mutations at a much faster rate than 
              healthy cells do. This leads to the emergence of an evolutionary process 
              inside the tumour. 
              A current research frontier is the characterization of the evolutionary 
              dynamics and phylogenies within individual cancer tumours, where multiple sub-populations of 
              cancer cells acquire differentiating sets of mutations. 
              </p>
              <p><span class="glyphicon glyphicon-file"></span>&nbsp;&nbsp;<a href="http://www.nature.com/nmeth/journal/v13/n7/full/nmeth.3867.html">Nature Methods paper on the analysis of single cell data</a></p>
              <p><span class="glyphicon glyphicon-plus"></span>&nbsp;&nbsp;<a href="https://www.stat.ubc.ca/~bouchard/papers.html#Evolutionary%20dynamics%20and%20phylogeny%20of%20cancer">More papers</a></p>
            </div> 
          </div>
          <div class="row marketing">
            <div class="col-lg-4">
              <img src="./larkz_files/phonemes.jpg" class="img-rounded img-responsive" style="max-width: 200px;">
            </div>
            <div class="col-lg-8">
              <h4>Computational historical linguistics</h4>
              <p>Phylogenetic trees (or networks, forests, etc) also play an important in 
              linguistics, to describe how language change and splits in ancestral speaker 
              populations gave rise to today's linguistic diversity. Computational methods are 
              also starting to play an important role in this field. 
              </p>
              <p><span class="glyphicon glyphicon-file"></span>&nbsp;&nbsp;<a href="https://www.stat.ubc.ca/~bouchard/pub/Bouchard2013Automated.pdf">PNAS paper on automated ancient language reconstruction</a></p>
              <p><span class="glyphicon glyphicon-plus"></span>&nbsp;&nbsp;<a href="https://www.stat.ubc.ca/~bouchard/papers.html#Computational%20historical%20linguistics">More papers</a></p>
            </div> 
          </div>
          <b>See also:</b> <a href="https://arxiv.org/find/stat/1/au:+Bouchard_Cote_A/0/1/0/all/0/1">my arXiv pre-prints</a>
        <p></p>
      </div>
      <div class="row marketing">
        <h3>Bio</h3>
        <hr>
        <p>
          </p><p id="bio">
            My main field of research is in computational statistics/statistical machine learning. 
            I am interested 
            in the mathematical side of the subject as well as in applications in linguistics and biology.
          </p>
          
          <p>
            On the methodology side, I am interested in Monte Carlo methods such as SMC and MCMC, 
            graphical models, non-parametric Bayesian statistics, randomized algorithms, and 
            variational inference.
          </p>
          
          <p>
            My favorite applications, both in linguistics and biology, are related to 
            phylogenetics in one way or another. Some examples of things I have currently/recently 
            been working on: automated reconstruction of proto-languages; cancer phylogenetics; 
            population genetics; pedigrees, tree and alignment inference.
          </p>
          
          <p>
            In the past, I also did some work on machine translation, on logical 
            characterization and approximation of labeled Markov processes, and on 
            reinforcement learning.
          </p>
          
          <h4>Academic background</h4>
          <ul>
            <li><b>University of California, Berkeley</b> (2005–2010).<ul>
              <li>PhD in Computer Science with a Designated Emphasis in Statistics. </li>
              <li>Minor in Linguistics.</li>
              <li>Research advisors: <a href="https://people.eecs.berkeley.edu/~jordan/">Michael I. Jordan</a> and <a href="https://people.eecs.berkeley.edu/~klein/">Dan Klein</a>. </li>
              <li>Title of dissertation: Probabilistic Models of Evolution and Language Change. </li>
            </ul>
            </li>
            <li><b>McGill University</b> (2002–2005).</li>
            <ul>
              <li>Joint Honours Program in Mathematics and Computer Science. </li>
              <li>Research advisors: <a href="http://cs.mcgill.ca/~dprecup/">Doina Precup</a> and <a href="http://www.cs.mcgill.ca/~prakash/">Prakash Panangaden</a>.</li>
            </ul>
          </ul>
          <h4>Employment</h4>
          <ul>
            <li><b>University of British Columbia</b> (2011–Present).</li>
            <ul>
              <li>Associate Professor of Statistics (2016–Present)</li>
              <li>Associate Member of the Computer Science Department (2014–Present).</li>
              <li>Assistant Professor of Statistics (2011–2016)</li>
            </ul>
          </ul>
          <h4>Awards</h4>
          <ul>
            <li><b>PIMS-UBC Mathematical Sciences Young Faculty Award</b> (2018)</li>
            <li><b>Tweedie Award</b> (2016)</li>
            <li><b>Google Faculty Award</b> (2013)</li>
            <li><b>Martha Piper Award</b> (2012)</li>
            <li><b>NIPS Best Student Paper Award</b> (2011)</li>
            <li><b>Siebel Scholar</b> (2010)</li>
            <li><b>Bourse du Fond de Recherche du Québec</b> (2010)</li>
            <li><b>NSERC Post-Graduate Scholarship</b> (2008)</li>
            <li><b>Courtemanche Scholarship</b> (2004)</li>
            <li><b>James McGill Scholarship</b> (2002)</li>
          </ul>
        <p></p>
      </div>
    
    </div>
    
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="./larkz_files/jquery.min.js"></script>
    <script>window.jQuery || document.write('<script src="https://www.stat.ubc.ca/~bouchard/assets/js/vendor/jquery.min.js"><\/script>')</script>
    <script src="./larkz_files/bootstrap.min.js"></script>
    
  

</body></html>